packages/graphback-codegen-offix/tests/OffixPluginTest.ts

```diff
@@ -0,0 +1,27 @@
+import { readFileSync } from 'fs';
+import test, { ExecutionContext } from 'ava';
+import { buildSchema, printSchema } from 'graphql';
+import { GraphbackCoreMetadata } from '@graphback/core';
+import { OffixPlugin } from '../src/OffixPlugin';
+
+const schemaText = readFileSync(`${__dirname}/mock.graphql`, 'utf8')
+
+test('Test snapshot config gql', async (t: ExecutionContext) => {
+  const defautConfig = {
+    "create": true,
+    "update": true,
+    "findAll": true,
+    "find": true,
+    "delete": true,
+    "subCreate": true,
+    "subUpdate": true,
+    "subDelete": true
+  }
+
+  const schemaGenerator = new OffixPlugin({ outputPath: './tmp' })
+  const metadata = new GraphbackCoreMetadata({
+    crudMethods: defautConfig
+  }, buildSchema(schemaText))
+  const schema = schemaGenerator.transformSchema(metadata)
```

Comment:
We are missing here GraphQL Schema extension. I wasn't sure if we want to add that as it will not longer be unit test. We have it already in dependencies so I guess we can extend that later to see how input types will be extended.